packages/autometrics-lib/src/buildInfo.ts

```diff
@@ -0,0 +1,40 @@
+import { getMeter } from "./instrumentation";
+
+type BuildInfo = {
+  version?: string;
+  commit?: string;
+  branch?: string;
+};
+
+let buildInfo: BuildInfo | undefined;
+
+export function registerBuildInfo() {
+  if (buildInfo) {
+    return;
+  } else {
+    buildInfo = {
+      version: getVersion(),
+      commit: getCommit(),
+      branch: getBranch(),
+    };
+
+    const gauge = getMeter().createUpDownCounter("build_info");
+    gauge.add(1, buildInfo);
+  }
```

Comment:
As you're already returning in the `if` statement above you can omit the explicit `else` syntax.